README.md

```diff
@@ -1,186 +1,50 @@
-
-### Open Edition
-
-An open edition auction requires the offering of a Master Edition NFT that specifically has no set supply. The auction will only create Prints of this item for bidders: each bidder is guaranteed to get a print, as there are no true "winners" of this auction type.
-
-An open edition auction can either have a set fixed price (equivalent to a Buy Now sale), can be set to the bid price (Pay what you want), or can be free (Make any bid to get it for free).
-
-### Limited Edition
-
-For a limited edition auction, a Master Edition NFT (of limited or unlimited supply) may be provided to the auction with a number of copies as the set amount of winning places.
-
-For each prize place, a Print will be minted in order of prize place, and awarded to the winning bidder of that place.
-
-For example, the first place winner will win Print #1; the second place winner Print #2; and so on.
-
-It is required for limited supply NFTs that there is atleast as much supply remaining as there are desired winners in the auction.
-
-### Tiered Auction
-
-A tiered auction can contain a mix of the other three auction types as winning placements. For instance, the first place winner could win a Print of Limited Edition NFT A, while the second-place winner could win a Normal NFT, etc. Additionally, all participants who did not win any place could get a Participation NFT Print from a Master Edition (if the Master Edition had no supply limit).
-
-## Royalties
-
-Metaplex can seamlessly create on-chain artist splits that remove the awkwardness out of collaboration.
-
-Tag each collaborator, set custom percentages, and you’re off to the races. Each NFT can also be minted with configurable royalty payments that are then sent automatically back to the original creators whenever an artwork is resold on a Metaplex marketplace in the future.
-
-## Storefronts
-
-Metaplex's off-chain component allows creators to launch a custom storefront, similar to Shopify or WordPress. This open-source project provides a graphical interface to the on-chain Metaplex program, for creators, buyers, and curators of NFTs. The design and layout of storefronts can be customized to suit the needs of the entity creating it, either as a permanent storefront or an auction hub for a specific auction or collection.
-
-All identification on the Storefront is based on wallet addresses. Creators and store admins sign through their wallets, and users place bids from connected wallets. Custom storefronts allow creators to create unique experiences per auction. Additionally, the Metaplex Foundation is working on multiple partnerships that will enable building immersive storefronts using VR/AR.
-
-## Development
-
-### Testing
-
-Testing changes to a rust program
-
-```
-# Change to devnet
-solana config set --url devnet
-
-# Build the project (takes a few mins)
-cd rust
-cargo build-bpf
+## SDKs
```

Comment:
We should add a note somewhere about the low-level autogenerated SDKs and where they can be found.

Reply:
Agreed. We could maybe add another section called "Autogenerated Libraries" (we should try and avoid calling them SDKs to avoid confusion and they are not really Developer Kits IMO) after the "SDKs" section.

Something like:

```md
## SDKs
...

## Autogenerated Libraries

- [MPL Token Metadata](https://github.com/metaplex-foundation/metaplex-program-library/tree/master/token-metadata/js)
- [MPL Candy Machine](https://github.com/metaplex-foundation/metaplex-program-library/tree/master/candy-machine/js)
- [MPL Auction House](https://github.com/metaplex-foundation/metaplex-program-library/tree/master/auction-house/js)
- ...
```

Reply:
Perhaps we should also link to the npm packages as well.

Reply:
Yes, good shout!

Reply:
How about the following format?

---

## Autogenerated Libraries

- **MPL Token Metadata.** [GitHub](https://github.com/metaplex-foundation/metaplex-program-library/tree/master/token-metadata/js) / [NPM](TODO)
  Some description here Elit laborum ad deserunt eiusmod veniam Lorem dolor velit.
- **MPL Candy Machine.** [GitHub](https://github.com/metaplex-foundation/metaplex-program-library/tree/master/candy-machine/js) / [NPM](TODO)
  Some description here tempor nulla tempor dolore nulla.
- **MPL Auction House.** [GitHub](https://github.com/metaplex-foundation/metaplex-program-library/tree/master/auction-house/js) / [NPM](TODO)
  Some description here Nisi eu proident eu reprehenderit ad consequat adipisicing nisi consectetur velit culpa dolore.
- ...

Reply:
love it! Would there be any way to bundle the rust crates in that last bit somewhere?

Reply:
I updated it! It should be better now? I think we could definitely try and add some cool images and stuff and maybe some emoji lol. Idk